src/hotspot/os/aix/libperfstat_aix.cpp

```diff
@@ -1,4 +1,5 @@
 /*
+ * Copyright (c) 2022, 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Yes there is, but it might not be a correct one as David's and your comments seem to indicate :smirk: 

The thought was that as a member of a company that has signed the OCA, work I do falls under Oracle's copyright. So any files I modify ought to have their copyright years updated. When there wasn't an Oracle copyright line, I added one (this may have been the problematic choice). Any guidance on how to do this properly is appreciated.